stock_request_analytic/models/analytic.py

```diff
@@ -1,29 +0,0 @@
-# Copyright 2017-2020 ForgeFlow, S.L. (https://www.forgeflow.com)
-# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
-
-from odoo import fields, models
-
-
-class AccountAnalyticAccount(models.Model):
-    _inherit = "account.analytic.account"
-
-    stock_request_ids = fields.One2many(
```

Comment:
Why removing this? I think it's still useful, but it should be computed and extracted from the analytic distributions.

Reply:
The analytical distribution is done at each record (in each `stock.request`), not at the analytical account level, so I can't extract it from there directly. Although what you mention could be done, I don't know if it is worthwhile and/or useful because I think this is not done in any other module.

Reply:
You should know if the smart-button was used or not, but I would bet it was...